conf/application.conf

```diff
@@ -63,6 +63,15 @@ play.modules {
   #disabled += ""
 }
 
+play.modules.enabled += io.grpc.examples.helloworld.GreeterServiceModule
```

Comment:
It would be cool to be able to avoid this.

We could enable an `akka.grpc.GrpcModule` from the `akka-grpc-runtime` `reference.conf`, but even then I'm not sure how we could enumerate all possible clients from there, since we cannot assume all gRPC code is generated from the same project: it might have been generated in a dependency.